wallet/how-to/manage-networks/use-multichain.md

```diff
@@ -0,0 +1,248 @@
+---
+description: Use the Multichain API.
+toc_max_heading_level: 4
+sidebar_custom_props:
+  flask_only: true
+---
+
+# Interact with multiple networks
+
+:::flaskOnly
+:::
+
+You can use the Multichain API to interact with multiple blockchain networks in MetaMask simultaneously.
+The API allows you to target specific chains as part of each method call, eliminating the need to
+detect and switch networks before executing signatures and transactions.
+
+:::note See also
+- [About the Multichain API](../../concepts/multichain-api.md)
+- [Multichain API reference](../../reference/multichain-api.md)
+:::
+
+## Prerequisites
+
+[Install MetaMask Flask.](/snaps/get-started/install-flask)
+
+## Steps
+
+### 1. Set up your project
+
+Establish a connection to MetaMask Flask and set up basic message handling using the
+[`wallet_notify`](../../reference/multichain-api.md#wallet_notify) event:
+
+```javascript
+// Initialize the connection to Flask.
+const EXTENSION_ID = "ljfoeinjpaedjfecbmggjgodbgkmjkjk"; // Replace this with ID of your Flask extension.
```

Comment:
Turns out that these IDs are canonical to the chrome web store so everyone gets the same ID

```suggestion
const EXTENSION_ID = "ljfoeinjpaedjfecbmggjgodbgkmjkjk"; // Flask extension ID
```